src/types/common.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface IMusicAccidental {
```

Comment:
Not a fan of the `I`-prefix convention. Let's use: `Accidental`, `NoteParts`, and `NoteValue` here please.